services/githubService.js

```diff
@@ -214,12 +225,42 @@ const fetchClosedIssues = async (params = {}) => {
   }
 };
 
+/**
+ * Fetches issues across all repositories in the ORG
+ */
+const fetchIssues = async () => {
+  try {
+    const baseURL = config.get("githubApi.baseUrl");
+    const issues = "/issues";
+    const urlObj = new URL(baseURL);
+    urlObj.pathname = "orgs" + "/" + config.get("githubApi.org") + issues;
+    const createdURL = urlObj.href;
+    const res = await getFetchWithAuthToken(
+      createdURL,
+      {
+        filter: "all",
+      },
+      {
+        Accept: "application/vnd.github+json",
+        // TODO: replace <AUTH-TOKEN> with RDS org PAT
+        Authorization: `Bearer <AUTH-TOKEN>`,
```

Comment:
Are we getting auth token from config?

Reply:
A PAT token for the RDS org needs to be created. Raised a ticket for this: https://github.com/Real-Dev-Squad/todo-action-items/issues/137
Can be created by someone who has ownership access.

Reply:
We should also create a ticket to accept the PAT from the config and use it here

Reply:
https://github.com/Real-Dev-Squad/todo-action-items/issues/137
Created a subtask within the same task

https://github.com/Real-Dev-Squad/todo-action-items/issues/146

Reply:
Thank you 🚀